include/config/mousebind.h

```diff
@@ -3,6 +3,7 @@
 #define __LABWC_MOUSEBIND_H
 
 #include <wayland-util.h>
+#include <wlr/types/wlr_output_layout.h>
```

Comment:
I think it makes sense to create our own enum in this case which could then also contain an `LAB_DIRECTION_INVALID` as index 0.